quilt/tools/store.py

```diff
@@ -129,6 +129,8 @@ def get_contents(self):
         try:
             with open(self._path, 'r') as contents_file:
                 contents = json.load(contents_file)
+                # Make the top-level a GROUP node
+                contents[TYPE_KEY] = NodeType.GROUP.value
```

Comment:
Just move this to the end of the function, before `return` - I think that should take care of everything?